### ISSUE TITLE
[shadow-dom] [spec-bug] in asignedNodes description.

### ISSUE THREAD

**Issue by trusktr** [WICG/webcomponents#612]
Not sure what this is supposed to say for the [description of assignedNodes](https://w3c.github.io/webcomponents/spec/shadow/#widl-HTMLSlotElement-assignedNodes-sequence-Node--AssignedNodesOptions-options):

![screen shot 2016-11-24 at 3 07 58 am](https://cloud.githubusercontent.com/assets/297678/20661651/ef275b82-b51d-11e6-859c-400c758929fb.png)

**Comment by annevk** [comment]
The spec is at https://dom.spec.whatwg.org/.

**Comment by hayatoito** [comment]
Yeah, let me close this. This section would be overridden with DOM Standard in the near future.
I will work on that.

**Comment by trusktr** [comment]
@annevk @hayatoito 

I just looked at https://dom.spec.whatwg.org, and searching for `assignedNodes` unfortunately led to zero results.

The most helpful spec is https://www.w3.org/TR/2015/WD-shadow-dom-20151215, which has useful tables and diagrams clearly explaining how ShadowDOM works, but no such thing can be found at https://dom.spec.whatwg.org.

TLDR,the new docs are not as useful as the old ones, and this is not a helpful situation for the new documentation. Many of that content can be easily copied over though.

I also think moving everything into one gargantuan megadoc is not the best idea. Maybe https://dom.spec.whatwg.org should link to different sub-docs such as https://www.w3.org/TR/2015/WD-shadow-dom-20151215.

At the very least, maybe https://dom.spec.whatwg.org should contain what it currently does, but with links to https://www.w3.org/TR/2015/WD-shadow-dom-20151215 and other sub-specs as necessary. This would make the documentation much better.

**Comment by hayatoito** [comment]
It is defined in HTML Standard: https://html.spec.whatwg.org/#dom-slot-assignednodes
